dotnet/src/Skills/Skills.Core/HttpSkill.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.SemanticKernel.Skills.Core;
 /// <example>
 /// Usage: kernel.ImportSkill("http", new HttpSkill());
 /// Examples:
-/// SKContext["url"] = "https://www.bing.com"
+/// SKcontext.Variables["url"] = "https://www.bing.com"
```

Comment:
I don't understand the casing change here.  Is there something that's actually cased as "SKcontext"?

Reply:
Nope, just a typo in the comments. Will fix